src/main/scala/org/ergoplatform/local/ErgoMiner.scala

```diff
@@ -317,13 +318,14 @@ class ErgoMiner(ergoSettings: ErgoSettings,
     //only transactions valid from against the current utxo state we take from the mem pool
     val emissionTxOpt = ErgoMiner.collectEmission(state, minerPk, ergoSettings.chainSettings.emissionRules).map { tx =>
       implicit val verifier: ErgoInterpreter = ErgoInterpreter(state.stateContext.currentParameters)
-      val cost = state.validateWithCost(tx, Some(upcomingContext)).get
+      val cost = state.validateWithCost(tx, Some(upcomingContext), Int.MaxValue).get
```

Comment:
Why not use maxTransactionComplexity?

Reply:
We'd like to collect emission anyway, even if its complexity is too high.